src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using DurableTask.AzureStorage;
+using DurableTask.Core;
+using DurableTask.Core.History;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager : IDurableManager
+    {
+        private readonly AzureStorageOrchestrationService _orchestrationService;
+
+        private readonly TaskHubClient _client;
+
+        private const string TaskHubName = "DurableFunctionsHub";
+
+        public DurableManager(ISecretsManager secretsManager)
+        {
+            var connectionString = secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals("AzureWebJobsStorage", StringComparison.OrdinalIgnoreCase)).Value;
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new CliException("Unable to retrieve storage connection string.");
+            }
+
+            var settings = new AzureStorageOrchestrationServiceSettings
+            {
+                TaskHubName = TaskHubName,
+                StorageConnectionString = connectionString,
+            };
+
+            _orchestrationService = new AzureStorageOrchestrationService(settings);
+            _client = new TaskHubClient(_orchestrationService);
+        }
+
+        public async Task DeleteHistory()
+        {
+            await _orchestrationService.DeleteAsync();
+            ColoredConsole.Write(Green("History and instance tables and queues successfully deleted."));
+        }
+
+        public async Task GetHistory(string instanceId)
+        {
+            var historyString = await _orchestrationService.GetOrchestrationHistoryAsync(instanceId, null);
+
+            JArray history = JArray.Parse(historyString);
+
+            JArray chronological_history = new JArray(history.OrderBy(obj => (string)obj["TimeStamp"]));
+            foreach (JObject jobj in chronological_history)
+            {
+                var parsed = Enum.TryParse(jobj["EventType"].ToString(), out EventType eventName);
```

Comment:
Let's add a comment explaining that we're converting the numeric enum values to the equivalent string values.